lasagne/init.py

```diff
@@ -50,6 +106,45 @@ def sample(self, shape):
 
 
 class Glorot(Initializer):
+    """Glorot weight initialization [1]_
+
+    This is also known as Xavier initialization.
+
+    Parameters
+    ----------
+    initializer : lasagne.init.Initializer
+        Initializer used to sample the weights. The default is Gaussian.
```

Comment:
If no one complains i'll remove the defaults and make a comment in the doc that people should use the *Normal and *Uniform shortcuts.